tests/torch/features/test_sequential.py

```diff
@@ -208,3 +208,46 @@ def test_sequential_and_non_sequential_tabular_features(yoochoose_schema, torch_
     outputs = tab_module(torch_yoochoose_like)
 
     assert list(outputs.shape) == [100, 20, 203]
+
+
+def test_sequential_tabular_features_with_text_module(yoochoose_schema, torch_yoochoose_like):
+    from typing import List
+
+    import torch
+
+    from transformers4rec.torch.features.base import InputBlock
+
+    class NormalizeTextembeddings(InputBlock):
```

Comment:
can we add one liner docstring to define what this class do?  to explain why text embeddings are required to be normalized?  we expect the user to define such class when they want to use the text embeddings right?

Reply:
I added a docstring explaining what the class is doing. The objective of this unit test is to showcase how to add a custom text-embeddings module to the T4Rec input block. So it assumes the user will define his custom `Textembeddings(InputBlock)` based on his dataset and his use-case.